backend/src/entryPoint.sh

```diff
@@ -9,5 +9,5 @@ python manage.py graph_models emissions -a -o /home/python/app/assets/database_s
 #python manage.py populate_data
 python manage.py collectstatic --noinput
 python manage.py check --deploy
-#python manage.py runserver 0.0.0.0:8000
-gunicorn -b 0.0.0.0:8000 pledge4future.wsgi
+python manage.py runserver 0.0.0.0:8000
```

Comment:
Can we maybe adjust this according to a flag set in the .env file? Some similar approach to the development flag in the .env.local file?

Reply:
good point, I've adapted it so it uses the `DJANGO_DEBUG` variable from the `.env` file to use the dev or production server.